src/Sarif.Viewer.VisualStudio/LoadSarifLogService.cs

```diff
@@ -39,9 +39,16 @@ public void LoadSarifLogs(IEnumerable<string> paths, bool promptOnSchemaUpgrade)
                 return;
             }
 
+            var cleanErrors = true;
             foreach (string path in paths)
             {
-                this.LoadSarifLog(path, promptOnSchemaUpgrade);
+                if (string.IsNullOrWhiteSpace(path))
```

Comment:
You could make this part of the filter by using a where

Reply:
Yep, I'll make this change